### DISCUSSION TITLE
Scheduled Keptn Service Execution

### DISCUSSION THREAD

**Discussion by ghost** [keptn/keptn#5146] [Ideas]
Related to #5144, it'd be beneficial if Keptn services could be triggered on a schedule. Aka a Keptn CronJob

**Comment by johannes-b** [comment]
Hello @Dynatrace-Adam-Gardner 

today I had a similar discussion with @thschue about a Keptn-service that triggers tasks based on a schedule (let's say every second week). Maybe there are some synergies between your idea of the CronJob and Thomas' approach of a schedule-based task (sequence) executor.
